open_humans/forms.py

```diff
@@ -81,7 +80,7 @@ class ChangePasswordForm(AllauthChangePasswordForm):
     A subclass of account's ChangePasswordForm that checks password length.
     """
 
-    def clean_password_new(self):
+    def clean_password(self):
```

Comment:
Sorry for the slow response on the PR. I started looking at the whole `_clean_password()` function here -- it was created to add some very basic "password strength" validation (i.e. a minimum length). This all seems to be obsolete.

1. It doesn't seem to be working? It's supposed to be imposing 8 characters minimum, and it's doing 6.
2. 6 is allauth's default in the absence of a password strength validator (a feature introduced in Django 1.9), see: https://github.com/pennersr/django-allauth/blob/master/allauth/account/app_settings.py#L147
3. We can set `ACCOUNT_PASSWORD_MIN_LENGTH = 8` and fix it to 8. This means adding three characters to this line (`LEN` -> `LENGTH`): https://github.com/OpenHumans/open-humans/blob/master/open_humans/settings.py#L364
4. And you can remove all this `_clean_password` override stuff that wasn't happening anyway - more code cleaned up :+1: